### PR TITLE
Feature/DAO creation overview page DAO-648

### PR DESCRIPTION
**Pull request by RakeshUP**

### CODE REVIEW

File: packages/web-app/src/containers/daoOverview/index.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+import {ButtonText, IconChevronRight} from '@aragon/ui-components';
+import CardWithImage from 'components/cardWithImage';
+import {useFormStep} from 'components/fullScreenStepper';
+import SelectBlockchain from '../../public/selectBlockchain.svg';
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/daoOverview/index.tsx]: You should also be able to use the import like so 'public/abc.svg' I believe...

File: packages/web-app/src/components/fullScreenStepper/fullScreenStepper.tsx [hunk 2]
```diff
@@ -57,19 +61,24 @@ export const FullScreenStepper: React.FC<FullScreenStepperProps> = ({
     wizardTitle,
     wizardDescription,
     hideWizard,
+    fullWidth,
```
Comment (Fabricevladimir) [inline packages/web-app/src/components/fullScreenStepper/fullScreenStepper.tsx:64]: I like the custom header and footer props

File: packages/web-app/src/locales/en/common.json [hunk 3]
```diff
@@ -145,5 +145,32 @@
         "title": "Select a UTC Timezone"
       }
     }
+  },
+  "createDAO": {
+    "title": "Create your DAO",
+    "overview": {
+      "title": "Get started with your DAO",
+      "description": "Create your own DAO in 4 easy steps + [TBD more helptext] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum scelerisque quam a quam molestie, vel euismod massa maximus. Duis mattis justo risus, nec sollicitudin leo mollis ac. Nulla orci tellus, hendrerit eu metus eget, dictum placerat sapien."
+    },
```
Comment (Fabricevladimir) [inline packages/web-app/src/locales/en/common.json:154]: Thanks for adding these labels

**Review by Fabricevladimir — APPROVED** [review]
Looks good! Thanks.

File: packages/web-app/src/pages/createDAO.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        <Step
+          hideWizard
+          fullWidth
+          customHeader={<OverviewDAOHeader />}
+          customFooter={<OverviewDAOFooter />}
+        >
+          <OverviewDAOStep />
+        </Step>
+        <Step
+          wizardTitle="Select Blockchain"
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/createDAO.tsx:32]: translation strings? :yum:

**Review by WissenIstNacht — APPROVED** [review]
Looks good to me, thanks! :)

File: packages/web-app/src/pages/createDAO.tsx (hunk 4 above)
Comment (RakeshUP) [reply packages/web-app/src/pages/createDAO.tsx:32]: Left it there to be done when the next step is done

**Pull request merged**